### ISSUE TITLE
Export mask_rcnn to Int8 Model (Floating Point exception

### ISSUE THREAD

**Issue by elb3k** [facebookresearch/d2go#81]
I was using Export model to Int8 Model using from `demo/d2go_beginner.ipynb` for mask_rcnn model, exact code:
```python
import copy
from detectron2.data import build_detection_test_loader
from d2go.model_zoo import model_zoo
from d2go.export.api import convert_and_export_predictor
from d2go.utils.testing.data_loader_helper import create_fake_detection_data_loader
from d2go.export.d2_meta_arch import patch_d2_meta_arch

import logging

# disable all the warnings
previous_level = logging.root.manager.disable
logging.disable(logging.INFO)

patch_d2_meta_arch()

cfg_name = 'mask_rcnn_fbnetv3a_dsmask_C4.yaml'
pytorch_model = model_zoo.get(cfg_name, trained=True)
pytorch_model.cpu()

with create_fake_detection_data_loader(224, 320, is_train=False) as data_loader:
    predictor_path = convert_and_export_predictor(
            model_zoo.get_config(cfg_name),
            copy.deepcopy(pytorch_model),
            "torchscript_int8@tracing",
            './',
            data_loader,
        )

# recover the logging level
logging.disable(previous_level)

```

Then python gives `terminated by signal SIGFPE (Floating point exception)`

Before the error, there were no other error, but these warnings:
`Arguments ['width_divisor', 'dw_skip_bnrelu', 'zero_last_bn_gamma'] skipped for op Conv2d
Some model parameters or buffers are not found in the checkpoint:
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_0.dw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_0.pw.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_0.pw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_0.pwl.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_0.pwl.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_1.dw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_1.pw.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_1.pw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_1.pwl.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_1.pwl.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_2.dw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_2.pw.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_2.pw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_2.pwl.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_2.pwl.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_3.dw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_3.pw.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_3.pw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_3.pwl.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_3.pwl.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_4.dw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_4.pw.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_4.pw.conv.weight
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_4.pwl.bn.{bias, running_mean, running_var, weight}
roi_heads.mask_head.feature_extractor.0.fbnetv2_0_4.pwl.conv.weight
roi_heads.mask_head.predictor.mask_fcn_logits.{bias, weight}
The checkpoint state_dict contains keys that are not used by the model:
  pixel_mean
  pixel_std
  proposal_generator.anchor_generator.cell_anchors.0
  roi_heads.mask_head.mask_fcn1.weight
  roi_heads.mask_head.mask_fcn1.norm.{bias, num_batches_tracked, running_mean, running_var, weight}
  roi_heads.mask_head.mask_fcn2.weight
  roi_heads.mask_head.mask_fcn2.norm.{bias, num_batches_tracked, running_mean, running_var, weight}
  roi_heads.mask_head.mask_fcn3.weight
  roi_heads.mask_head.mask_fcn3.norm.{bias, num_batches_tracked, running_mean, running_var, weight}
  roi_heads.mask_head.mask_fcn4.weight
  roi_heads.mask_head.mask_fcn4.norm.{bias, num_batches_tracked, running_mean, running_var, weight}
  roi_heads.mask_head.deconv.{bias, weight}
  roi_heads.mask_head.predictor.{bias, weight}
loading annotations into memory...
Done (t=0.00s)
creating index...
index created!
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/torch/quantization/observer.py:121: UserWarning: Please use quant_min and quant_max to specify the range for observers.                     reduce_range will be deprecated in a future release of PyTorch.
  warnings.warn(
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/torch/quantization/observer.py:955: UserWarning: must run observer before calling calculate_qparams.                                    Returning default scale and zero point 
  warnings.warn(
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/structures/image_list.py:92: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert t.shape[:-2] == tensors[0].shape[:-2], t.shape
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/structures/boxes.py:158: TracerWarning: torch.as_tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
  tensor = torch.as_tensor(tensor, dtype=torch.float32, device=device)
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/structures/boxes.py:163: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert tensor.dim() == 2 and tensor.size(-1) == 4, tensor.size()
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/modeling/proposal_generator/proposal_utils.py:101: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if not valid_mask.all():
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/structures/boxes.py:200: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert torch.isfinite(self.tensor).all(), "Box tensor contains infinite or NaN!"
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/torch/tensor.py:587: RuntimeWarning: Iterating over a tensor might cause the trace to be incorrect. Passing a tensor of different shape won't change the number of iterations executed (and might lead to errors or silently give incorrect results).
  warnings.warn('Iterating over a tensor might cause the trace to be incorrect. '
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/layers/nms.py:25: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert boxes.shape[-1] == 4
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/modeling/poolers.py:216: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert len(box_lists) == x[0].size(
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/layers/roi_align.py:55: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert rois.dim() == 2 and rois.size(1) == 5
/home/elbek/Projects/playground/test/d2go/d2go/modeling/backbone/fbnet_v2.py:372: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if len(x.shape) == 4 and (x.shape[2] > 1 or x.shape[3] > 1):
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/modeling/roi_heads/fast_rcnn.py:138: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if not valid_mask.all():
/home/elbek/anaconda3/envs/d2go/lib/python3.8/site-packages/detectron2/modeling/roi_heads/fast_rcnn.py:155: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if num_bbox_reg_classes == 1:`

**Comment by zhanghang1989** [comment]
We have updated the model configs and pretrained weights. Could you try again with the latest code?
